src/llvm/builder.cr

```diff
@@ -387,6 +387,12 @@ class LLVM::Builder
     LibLLVM.dispose_builder(@unwrap)
   end
 
+  def finalize
+    return if @disposed
```

Comment:
**suggestion:** This seems redundant because `dispose` already checks if it's already been disposed before.

Reply:
yea, upon looking back at the code again, it is. Will remove